src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis</RootNamespace>
-    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetVS);netstandard2.0</TargetFrameworks>
```

Comment:
It's unclear to me why we're multi-targeting EditorFeatures -- we don't load that in OOP as far as I can tell. This would have been loaded in VS for Mac but that's not something we're targeting anymore.

Reply:
One benefit is nullable reference types for the framework assemblies.

Reply:
Oh right! It'd almost be nice then if we had a separate variable for that reason either just to self-document or so that way it's clear we can target new TFMs there almost immediately -- i.e. there's no reason we couldn't already use net9.0 once we adopt a 9.0 SDK for the first time.